components/event-header.vue

```diff
@@ -0,0 +1,81 @@
+<template>
```

Comment:
This file is currently not in a folder the same way all other components are stored (e.g. `event-header/EventHeader.vue`). Maybe also put it in a folder for now?

Reply:
Done in dd99fb6d